neurobooth_os/tests/test_logging.py

```diff
@@ -1,19 +1,62 @@
+import inspect
 import logging
 import os
 import shutil
 import sys
 import unittest
 
-from neurobooth_os.log_manager import make_default_logger
+from neurobooth_terra import Table
+
+from neurobooth_os.log_manager import make_default_logger, make_db_logger, _test_log_handler_fallback
+from neurobooth_os.iout.metadator import get_conn
 
 log_path = r"C:\neurobooth\test_data\test_logs"
-logger = make_default_logger(log_path)
+database = "mock_neurobooth_1"
+connection = None
+
+subject = "1111111"
```

Comment:
Maybe change this to `000000`, as this would more clearly be a test ID that would be caught by existing hacky rules if they are implemented anywhere.

Reply:
I made the change, but these tests should _never_ be run pointing at the production database (as that would delete all the logs during cleanup).  In general, it's bad practice to put system test data into a production database as it has the potential to adversely impact any downstream analysis of the data.